exasol/ds/sandbox/runtime/ansible/roles/entrypoint/files/entrypoint.py

```diff
@@ -0,0 +1,90 @@
+import argparse
+import shutil
+import subprocess
+import time
+
+from inspect import cleandoc
+from pathlib import Path
+
+
+def arg_parser():
+    parser = argparse.ArgumentParser(
+        description="entry point for docker container",
+    )
+    parser.add_argument(
+        "--copy-from", action="append", type=Path, metavar="<SRC>",
+        help="""Copy files recursively with permissions.
+        For each option --copy-from there must be corresponding option --copy-to.""",
+    )
+    parser.add_argument(
+        "--copy-to", action="append", type=Path, metavar="<DEST>",
+        help="destination location for file to copy",
+    )
+    parser.add_argument(
+        "--jupyter-server", action="store_true",
+        help="start server for Jupyter notebooks",
+    )
+    parser.add_argument(
+        "--sleep", action="store_true",
+        help="keep script running infinitly",
+    )
+    return parser
+
+
+def start_jupyter_server():
+    subprocess.run([
+        "/root/jupyterenv/bin/jupyter-lab",
+        "--notebook-dir=/root/notebooks",
+        "--no-browser",
+        "--allow-root",
+    ])
+    print(cleandoc(
+        """
+        Server for Jupyter has been started.
+        You can connect with https://localhost:8888
+        """))
+
+
+def copy_rec(src: Path, dst: Path):
+    """
+    Copy files and directories missing in dst from src and set
+    permission 644 for files and 755 for directories.
+    If directory src does not exit then do not copy anything.
+    """
```

Comment:
Yes, sure, I will update the implementation.

Reply:
NameError: name 'top' is not defined

Reply:
OK `pathlib.Path.walk()` is available only since python 3.12.
I used o.walk() instead. 
Still I liked the old implementation better as it was ~ 50%  shorter and used fewer libs.